site/docs/pages/reference/datatypes/events.md

```diff
@@ -8,11 +8,14 @@ Snapchain keeps events around for 3 days after which they are deleted to save sp
+| Field        | Type                                                                                                                                                                                                                                                                                                                                       | Label | Description                                                       |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----- | ----------------------------------------------------------------- |
+| type         | [HubEventType](#HubEventType)                                                                                                                                                                                                                                                                                                              |       | The type of event                                                 |
+| id           | [uint64](#uint64)                                                                                                                                                                                                                                                                                                                          |       | Unique identifier for the event that encodes block height ordering |
+| block_number | [uint64](#uint64)                                                                                                                                                                                                                                                                                                                          |       | The block number when the event was created                      |
+| shard_index  | [uint32](#uint32)                                                                                                                                                                                                                                                                                                                          |       | The shard index where the event occurred                          |
+| timestamp    | [uint64](#uint64)                                                                                                                                                                                                                                                                                                                          |       | The timestamp when the event was created                          |
```

Comment:
I would describe this as "The timestamp when the block that produced this event was created". The current description is a bit ambiguous/misleading.